### ISSUE TITLE
lab6

### ISSUE THREAD

**Issue by ruizt** [pstat197/pstat197a#9]
revise instructions in final task; training set vocabulary will differ slightly from test set, so need to elaborate on how to preprocess

**Comment by ruizt** [comment]
also consider replacing logistic regression example so that parameter estimates from glm match weights
